CHANGELOG

```diff
@@ -1,3 +1,127 @@
+== OpenDataPlane (1.19.0.1)
+=== Summary of Changes
+ODP v1.19.0.1 is the first service update for the Tiger Moth release. It
+incorporates a number of corrections and enhancements that improve the quality
+and testability of ODP.
+
+==== APIs
+There are no API changes in this release.
+
+==== DPDK Service Release Sync
+ODP now uses DPDK 17.11.2 for it's DPDK-related support. This is the current
+service level for the DPDK 17.11 LTS package used by ODP Tiger Moth and
+incorporates a number of important bug fixes.
+
+=== Implementation Improvements
+The ODP reference implementation has been improved in a number of areas.
+
+==== GCC 8 Support
+The GCC 8 series of compilers provides additional warnings about possible
+string truncation. A few ODP modules were changed to avoid misleading
+warnings when compiling with this level of GCC.
+
+==== Linking with `libnuma`
+The `libnuma` library is used for DPDK pktios to provide proper memory
+allocation on NUMA-aware systems. Linking with this library is improved to
+avoid extraneous error messages at build time.
+
+==== Packet metadata reorganization
```

Comment:
This is also a performance improvement (compare vs next section).

Reply:
Performance improvement is called out as one of the effects of this change. I don't have a separate performance section in this changelog so I'm not sure what you're suggesting here.

Reply:
@Bill-Fischofer-Linaro I'm suggesting to either rename later section to smth. like 'fdserver improvements' or to create separate 'performance' section.

Reply:
I see. Since `fdserver` is purely internal perhaps rename that to `Shmem improvements` to relate it to user-visible function? That's its main use/effect.

Reply:
Ues, that's fine with me.